src/components/panel/panel.js

```diff
@@ -1679,18 +1687,33 @@ MdPanelRef.prototype._addStyles = function() {
       return; // Don't setup positioning.
     }
 
-    // Wait for angular to finish processing the template, then position it
-    // correctly. This is necessary so that the panel will have a defined height
-    // and width.
+    // Wait for angular to finish processing the template
     self._$rootScope['$$postDigest'](function() {
+      // Position it correctly. This is necessary so that the panel will have a
+      // defined height and width.
       self._updatePosition(true);
+
+      // Theme the element and container.
+      self._setTheming();
+
       resolve(self);
     });
   });
 };
 
 
 /**
+ * Sets the `$mdTheming` classes on the `panelContainer` and `panelEl`.
+ * @method setTheming
```

Comment:
This line isn't needed, `@method` is only used for the ngdoc, but this isn't public.